scripts/install.sh

```
@@ -0,0 +1,522 @@
+#!/bin/bash
+
+set -e
```

Consider adding `set -eu` as well to avoid silent errors (also `set -o pipefail` potentially although probably not needed)